src/plugins/vis_augmenter/public/saved_augment_vis/saved_augment_vis.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { get, isEmpty } from 'lodash';
```

Comment:
Let;s not introduce more uses of `lodash`; we are actively trying to get rid of it.